examples/experimental/slpk-via-range-requests/src/custom-tile-3d-layer.ts

```diff
@@ -0,0 +1,41 @@
+import {Tile3DLayer, Tile3DLayerProps} from '@deck.gl/geo-layers';
+import {UpdateParameters} from '@deck.gl/core/typed';
+
+export default class CustomTile3DLayer<
```

Comment:
Please add comments explaining what this custom layer does, why we needed to subclass it. The switch to range requests should ideally not require any changes to the layer?